gammapy/utils/modeling.py

```diff
@@ -55,17 +59,34 @@ class UnknownModelError(ValueError):
 
 
 class Parameter(object):
+    """
+    Class representing model parameters.
+
+    Parameters
+    ----------
+    name : str
+        Name of the parameter.
+    value : float or `~astropy.units.quantity`
```

Comment:
Should this really be "value: float or quantity"?
Can we make it just float to keep it simple?